packages/styles/scss/components/content-item-horizontal-thumbnail/_content-item-horizontal-thumbnail.scss

```diff
@@ -0,0 +1,159 @@
+/**
```

Comment:
don't think we need to have a separate `scss` file in the styles package, can you move this to the `scss` file you created within the `content-item-horizontal` web-components folder?